src/text-fragment-utils.js

```diff
@@ -873,6 +873,9 @@ const isValidColor = (color) => {
  * @param {Object} - background-color and color that will be applied to the element style
  */
 export const setMarkStyle = (mark, {backgroundColor, color}) => {
-  if (backgroundColor) mark.style.backgroundColor = backgroundColor;
-  if (color) mark.style.color = color;
+  const cssRuleBackgroundColor = backgroundColor ? `background-color: ${backgroundColor};` : '';
+  const cssRuleColor = color ? `color: ${color};` : '';
+  mark.setAttribute('style', `${cssRuleBackgroundColor && cssRuleColor ?
```

Comment:
The formatting/wrapping looks a bit off; could you run `npm prepare` (or `npm install`) on this patch?

(If you've already done this then just let me know and we can merge as-is)